runtime/Cpp/runtime/src/TokenStreamRewriter.cpp

```diff
@@ -93,7 +93,7 @@ TokenStreamRewriter::TokenStreamRewriter(TokenStream *tokens_) : tokens(tokens_)
 }
 
 TokenStreamRewriter::~TokenStreamRewriter() {
-  for (auto program : _programs) {
+  for (const auto& program : _programs) {
```

Comment:
Coding convention: that should be: `for (const auto &program : _program) {`

Reply:
Missed this one, thanks.